MicroProfile-Rest-Client/tck-runner/pom.xml

```diff
@@ -198,6 +199,20 @@
                 </dependency>
             </dependencies>
         </profile>
+        <profile>
+            <id>payara5.194+</id>
+            <activation>
+                <property>
+                    <name>payara.version</name>
+                    <value>/5\.(194|[2-9]\d\d)(?:-SNAPSHOT|\.\d*)?/</value>
+                </property>
+            </activation>
+            <properties>
+                <microprofile.rest.client.tck.version>1.3.4</microprofile.rest.client.tck.version>
+                <microprofile.rest.client.api.version>1.3.4</microprofile.rest.client.api.version>
+                <jersey.version>2.29.1.payara-p1</jersey.version>
```

Comment:
Is this actually used anywhere...?

Reply:
I'm not a fan of that - I'd rather just duplicate the stuff personally rather than "layer" profiles which tie to specific versions.